ext/date/tests/strtotime3-64bit.phpt

```diff
@@ -45,7 +50,7 @@ foreach ($strs as $str) {
 
 ?>
 --EXPECT--
-bool(false)
+strtotime(): Argument #1 ($time) cannot be empty
```

Comment:
Again, this often comes from user input, so I disagree with this change.

Reply:
Will revert